dipy/core/profile.py

```diff
@@ -52,7 +53,8 @@ class Profiler:
 
     """
 
-    def __init__(self, call=None, *args):
+    @warning_for_keywords()
+    def __init__(self, *args, call=None):
```

Comment:
missing`*`.   I suppose, it should be `def __init__(self, *args, *, call=None):`

Reply:
This will give syntax error, actually. I can't add separator after *args.
Now what will happen is whatever positional args are passed after self, will be stored in *args and will not even pass to call.
1. This will force user to call the keyword args properly, if they want to specifically use the keyword args.
2. If they don't input, we will have default value either-ways.

Also i can't keep `def __init__(self, *, call=None, *args)`, this will give same syntax error and not allowed.

The only other possible solution i see is `def __init__(self, args=(), *, call=None)`

Reply:
both `*args` and `arg=()' will create tuple to store the extra pos arguments

Reply:
Finally, here, you just need to remove the decorators